tests/util/test_wb_logger.py

```diff
@@ -87,7 +87,11 @@ def finish(self):
 mock_wandb = MockWandb()
 
 
-@mock.patch.object(wandb, "__init__", mock_wandb.__init__)
+# we ignore the type below as one should technically not access the
+# __init__ method directly but only by creating an instance.
+
```

Comment:
Why the two line whitespace? This makes it unclear that the comment is for line 94.

Reply:
That was a stupid PEP rule but I agree, we should override that here.